core/unit_test/TestComplex.hpp

```diff
@@ -688,6 +687,40 @@ TEST(TEST_CATEGORY, complex_structured_bindings) {
   test.testit();
 }
 
+#define CHECK_COMPLEX(_value_, _real_, _imag_) \
+  static_assert(_value_.real() == _real_);     \
+  static_assert(_value_.imag() == _imag_);
+
+constexpr bool construction_is_constexpr() {
+  [[maybe_unused]] constexpr Kokkos::complex<double> from_single{1.2};
+  [[maybe_unused]] constexpr Kokkos::complex<double> from_both{1.2, 3.4};
+  [[maybe_unused]] constexpr Kokkos::complex<double> from_none{};
+
+  CHECK_COMPLEX(from_single, 1.2, 0.);
+  CHECK_COMPLEX(from_both, 1.2, 3.4);
+  CHECK_COMPLEX(from_none, 0., 0.);
+
+  [[maybe_unused]] constexpr auto from_copy_assign = from_both;
+  [[maybe_unused]] constexpr auto from_copy_constr(from_both);
+
+  CHECK_COMPLEX(from_copy_assign, 1.2, 3.4);
+  CHECK_COMPLEX(from_copy_constr, 1.2, 3.4);
+
+  [[maybe_unused]] constexpr auto from_move_assign = std::move(from_both);
```

Comment:
Same here, this uses move constructor, not move assignment operator

Reply:
Indeed, let me correct this.